Pear/Profile/ProfileViewController.swift

```diff
@@ -42,11 +57,21 @@ class ProfileViewController: UIViewController {
         navigationController?.navigationBar.barTintColor = .backgroundLightGreen
         navigationController?.navigationBar.shadowImage = UIImage() // Hide navigation bar bottom shadow
         navigationController?.navigationBar.titleTextAttributes = [
-            .font: UIFont.getFont(.medium, size: 24)
+            .font: UIFont._20CircularStdMedium,
+            .foregroundColor: UIColor.primaryText
```

Comment:
can you just do `.primaryText`?

Reply:
You have to explicitly state type `UIFont` since titleTextAtributes values are of type `Any` that way Swift can find the member.